### ISSUE TITLE
DB Schema creation problems

### ISSUE THREAD

**Issue by julmud** [julmud/phpDVDProfiler#7]
I've received a report on the Invelos forums for some problems when creating the DB structure from scratch in MySQL version 5.7.28:

> Ich hab übrigens versucht, die Datenbank von neu an aufzubauen und bin dabei bei dem Script auf folgende Probleme gestoßen (MySQL 5.7.28).
>
> Er wollte nicht, dass die Id null ist, wenn sie Primary Key sein soll.
> 
> Hab es von
>   id char(20) default null,
> auf
>   id char(20),
> geändert
> 
> Außerdem mochte er das Keyword "signed" nicht ("unsigned" war OK) und ich hab es entfernt.

(Both problems have to do with the _DVD table.)

I think I'd tried the full schema installation with MariaDB 10.4.10 without any problem, but I'll have to take a deeper look into that.

**Comment by gjnoonan** [comment]
@julmud If I have translated correctly, I had a similar issue when trying to run this on MySQL 8.0

```
All parts of a PRIMARY KEY must be NOT NULL; if you need NULL in a key, use UNIQUE instead
```

**Comment by julmud** [comment]
So, I thought that including "Fixes #7" in my commit message would automatically close the referenced issue when the branch gets merged on master... Obviously, I must be doing something wrong... :-/

**Comment by gjnoonan** [comment]
> So, I thought that including "Fixes #7" in my commit message would automatically close the referenced issue when the branch gets merged on master... Obviously, I must be doing something wrong... :-/

It is because you said `Fixes issue #7` it just needs to be `fixes #7` (or indeed `fixes: #7`)

The following keywords _should_ auto close the issue:
close
closes
closed
fix
fixes
fixed
resolve
resolves
resolved
